Formula/tuntox.rb

```diff
@@ -0,0 +1,43 @@
+class Tuntox < Formula
+  desc "Tunnel TCP connections over the Tox protocol"
+  homepage "https://gdr.name/tuntox/"
+  url "https://github.com/gjedeer/tuntox/archive/refs/tags/0.0.10.1.tar.gz"
+  sha256 "7f04ccf7789467ff5308ababbf24d44c300ca54f4035137f35f8e6cb2d779b12"
+  license "GPL-3.0-only"
+  head "https://github.com/gjedeer/tuntox.git", branch: "master"
+
+  depends_on "cscope" => :build
+  depends_on "pkg-config" => :build
+  depends_on "toxcore"
+
+  def install
+    if OS.mac?
+      inreplace "Makefile.mac", ".git/HEAD .git/index", ""
```

Comment:
Won't this lead to garbage in the version output?

Reply:
Yes

The git sha is placed in `gitversion.h`

Would a simple hardcoded sha via an `inreplace` be sufficient here?

Reply:
Let's just replace it with `"N/A"` to avoid needing a git clone and also having to manually modify the commit on version bumps.